internal/workflow/preprocessing_test.go

```diff
@@ -613,7 +613,7 @@ file format fmt/11 not allowed: "fake/path/to/sip/file2.png"`,
 				},
 				{
 					Name:        "Validate SIP metadata",
-					Message:     "Content error: metadata validation has failed:\nfake/path/to/sip/additional/UpdatedAreldaMetadata.xml does not match expected metadata requirements",
+					Message:     "Content error: metadata validation has failed:\n\n/additional/UpdatedAreldaMetadata.xml does not match expected metadata requirements",
```

Comment:
I think the extra linebreak is due to my comment on using `append()`.

Reply:
Yeah.